.github/workflows/tests.yml

```diff
@@ -33,7 +32,7 @@ jobs:
   ubuntu:
     strategy:
       matrix:
-        os: [ubuntu-18.04]
+        os: [ubuntu-latest]
```

Comment:
Could also be LTS supported version? Right now this is 20.4 and in April will become 22.4 until 2024

Reply:
In principle yes, but we use `-latest` for the other two OSes.

...okay, so Aesara uses `ubuntu-latest` then I'll change this to `ubuntu-20.04` so we get higher coverage overall